gremlin-go/docker/gremlin-server-integration-secure.yaml

```diff
@@ -20,6 +20,7 @@ port: 45941
 evaluationTimeout: 30000
 graphs: {
   graph: conf/tinkergraph-empty.properties,
+  test: conf/tinkergraph-empty.properties,
```

Comment:
what is this additional "test" configuration for?

Reply:
Cucumber and integration tests run in parallel, Cucumber was modifying the graph so we added an empty test graph to also use for testing in integration so that there wasn't race conditions causing intermittent failures with the parallel tests.